src/components/sidebar/ExpandButton.tsx

```diff
@@ -0,0 +1,26 @@
+import {
+  ArrowLeftOnRectangleIcon,
+  ArrowRightOnRectangleIcon,
+} from "@heroicons/react/24/solid";
+import { Button } from "@/components";
+
+interface ExpandButtonProps {
+  isOpen: boolean;
+  onClick: (status: boolean) => void;
+}
+
+export default function ExpandButton({ isOpen, onClick }: ExpandButtonProps) {
+  return (
+    <Button
+      title="Expand"
+      customClassName="w-[3.125rem] bg-white text-black capitalize hover:bg-white mr-2 border-none"
```

Comment:
change bg to bg-base-200 
change text to text-base-300
change hover to bg-base-200